iam/terraform/account-specific/main/main.tf

```diff
@@ -17,6 +17,6 @@ terraform {
   }
 
   required_providers {
-    aws = "5.22.0"
+    aws = "5.23.0"
```

Comment:
Why was this bumped to `5.23.0` when all the other terraform files were bumped to `5.23.1`?

Reply:
You are correct, will update it to 5.23.1